src/xdsl/dialects/memref.py

```diff
@@ -44,29 +44,29 @@ def alloc(self,
               return_type: Attribute,
               shape: List[int] = [1]) -> Operation:
         return Alloc.create(
-            [], [MemRefType.get(return_type, shape)],
+            [], [MemRefType.from_type_and_list(return_type, shape)],
```

Comment:
You do not use the builder interface here, as you know which exact builder method you want to call, right?

Reply:
Yes, though I'll probably change this in a next PR. For now, I just want it to work like before.